### PR TITLE
wuson

### PR DESCRIPTION
**Pull request by wsq815**
wuson Wu 自由职业者

### CODE REVIEW

**Comment by MSWorkers** [comment]
@wsq815 looks like you did the PR the other way. Please just tell me how you'd like to sign and i'll be happy to add it for you! 请告诉我你想如何签名，我为你添加！

**Comment by Particaly** [comment]
signed-J.S.Patrick

File: README.md [hunk 1]
```diff
@@ -51,227 +51,356 @@ To add your name to the list of supporters, make a pull request with your name a
...
+* Zhou Quan, Software Engineer, SmartRecruiters
+* Ziang Chen, Software Engineer
+* ZiHang Gao, Software Engineer, IReader
+* Zion Chen, Software Engineer, SAP
+* Ziyi Li(李子毅), Student
+* Zongbao Feng, Engineer, Baidu
+* Zora Tung, Software Engineer, Google
+* Zuokun Ouyang, PhD Student, Université d'Orléans
+* Xianli Si, Software Engineer, Belle International
+* Lang Qin, IC Verification Engineer, Globalfoundries
```
Comment (Particaly) [inline README.md]: * J.S.Patrick, Front-end engineer

**Comment by Tedko** [comment]
你pull request的base选错了吧？patch是别的pr的

**Pull request closed without merging**